### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by ShonChassel** [community/community#64613] [Programming Help]
### Select Topic Area

Product Feedback

### Body

Hey friends
I would love to receive feedback of my web I am currently working on
https://github.com/ShonChassel
Thank you all

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ShonChassel ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
